front/app/components/CustomFieldsForm/index.tsx

```diff
@@ -54,6 +51,10 @@ const CustomFieldsForm = ({
   initialFormData?: FormValues;
 }) => {
   const [isDisclaimerOpened, setIsDisclaimerOpened] = useState(false);
+  const [formValuesForDisclaimer, setFormValuesForDisclaimer] =
```

Comment:
Now, these values are only used for the disclaimer, so I renamed it to make it more clear. Otherwise we have two variables with the same name in the same scope in the submit handler